src/ExpressReceiver.spec.ts

```diff
@@ -245,24 +239,15 @@ describe('ExpressReceiver', () => {
       const result: any = {};
       const resp = buildResponseToVerify(result);
 
-      let error: string = '';
-      let warn: string = '';
-      const logger = {
-        error: (msg: string) => { error = msg; },
-        warn: (msg: string) => { warn = msg; },
-      } as any as Logger;
-
       const next = sinon.fake();
 
       // Act
-      const verifier = verifySignatureAndParseBody(logger, signingSecret);
+      const verifier = verifySignatureAndParseRawBody(noopLogger, signingSecret);
       await verifier(req, resp, next);
 
       // Assert
       assert.equal(result.code, 400);
       assert.equal(result.sent, true);
-      assert.equal(error, 'Failed to parse body as JSON data for content-type: undefined');
```

Comment:
any reason we're removing these assertions?

Reply:
I removed the code that did this.